sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -45,47 +45,99 @@ func init() {
 }
 
 func main() {
-	os.Exit(_main())
+	if err := _main(); err != nil {
+		log.L.Fatalf("%v", err)
+	}
 }
 
-func _main() int {
-	// Parse command-line arguments
+func _main() error {
+	// Command-line arguments
 	var (
 		containerID      string
 		source           string
 		containerdSocket string
 		namespace        string
 		superpowered     bool
-		pullImageOnly    bool
 	)
```

Comment:
I want to say change this to not use predeclared vars, but I can't come up with a good reason right now to make it otherwise (using `cli.Context.String()` and friends instead). I'm in favor of leaving this as-is for now (Sam and I have debated this before ourselves, I'm ambivalent).

Reply:
I like the predeclared vars.

Reply:
> I like the predeclared vars.

I know, but it's less clear in these circumstances because the two subcommands are each using them. There's not a lot of them and so the scope remains clear for now - I would push on this if there were more at play. For now, I think its reasonable and good to move forward with.